README.md

```diff
@@ -1,3 +1,24 @@
+JSHint - Fluid Project Edition
+==============================
+
+This repository contains the Fluid Project's version of the JSHint JavaScript linting tool.
+
+This version was forked from the main jshint project at its revision [`bf886b8f07124ddfbef158d7269e7b3b184a65fe`](https://github.com/jshint/jshint/commit/bf886b8f07124ddfbef158d7269e7b3b184a65fe)
+of Tue Mar 25 23:01:58 2014 -0700, which occurred shortly after the release of its 2.4.4 version, and before the
+release of its 2.5.0 version. Following that revision, all support for whitespace-based linting was removed
+from jshint.
+
+As well as retaining all features as of that revision, this version of the project includes the following
+extra facilities:
+
+ * When the `trailing:true` option is enabled, all lines containing trailing whitespace are flagged, including those that are otherwise blank or comment lines
+ * A new option `mulopwhite:true` can be enabled, which permits whitespace to be lax around multiplicative operators, allowing constructions like `2*x + 3*y` to be permissible
```

Comment:
I was playing with both online jshint http://jshint.com/ which should be using the latest jshint version, and the linting built into the infusion which uses jshint 2.4.0, none of them reports errors with either `2*x + 3*y` or `2 * x + 3 * y`. Did I miss something? What does `mulopwhite:true` actually do?